rasa/shared/importers/multi_project.py

```diff
@@ -183,18 +184,28 @@ async def get_stories(
         use_e2e: bool = False,
         exclusion_percentage: Optional[int] = None,
     ) -> StoryGraph:
-        story_paths = self._story_paths if not use_e2e else self._e2e_story_paths
+        """Retrieves training stories / rules (see parent class for full docstring)."""
+        if use_e2e:
+            return await self.get_conversation_tests()
 
         return await utils.story_graph_from_paths(
-            story_paths,
+            self._story_paths,
             await self.get_domain(),
             template_variables,
             use_e2e,
             exclusion_percentage,
         )
 
+    async def get_conversation_tests(self) -> StoryGraph:
+        """Retrieves conversation test stories (see parent class for full docstring)."""
+        return await utils.story_graph_from_paths(
+            self._e2e_story_paths, await self.get_domain(), use_e2e=True,
+        )
+
     async def get_config(self) -> Dict:
+        """Retrieves model config (see parent class for full docstring)."""
         return self.config
 
     async def get_nlu_data(self, language: Optional[Text] = "en") -> TrainingData:
+        """Retrieves NLU training data (see parent class for full docstring)."""
```

Comment:
Do we have a convention for docstrings on subclasses?

Reply:
Not specifically. It's somewhat my personal preference to write a proper docstring in one place and then link to it (ideally I'd just omit but that doesn't work with `pydocstyle`)